odoo/http.py

```diff
@@ -620,13 +620,17 @@ def _handle_exception(self, exception):
         try:
             return super(JsonRequest, self)._handle_exception(exception)
         except Exception:
-            if not isinstance(exception, (odoo.exceptions.Warning, SessionExpiredException,
-                                          odoo.exceptions.except_orm, werkzeug.exceptions.NotFound)):
-                _logger.exception("Exception during JSON request handling.")
+            if not isinstance(exception, SessionExpiredException) \
+                    and exception.args[:1] != "bus.Bus not available in test mode":
```

Comment:
`and exception.args[:1] != "bus.Bus not available in test mode":`

can't we just move it in the if and just do a logger.info ?